test/hotspot/jtreg/vmTestbase/nsk/jvmti/GetObjectMonitorUsage/objmonusage001.java

```diff
@@ -63,7 +63,7 @@ public static int run(String argv[], PrintStream out) {
         }
         // Virtual threads are not supported by GetObjectMonitorUsage.
         // Correct the expected values if the test is executed with
-        // JTREG_TEST_THREAD_FACTORY=Virtual.
+        // TEST_THREAD_FACTORY=Virtual.
```

Comment:
Perhaps these references should use the full `JTREG=TEST_THREAD_FACTORY=Virtual` to make it clearer.

Reply:
Thanks for the review, use `JTREG="JTREG_TEST_THREAD_FACTORY=Virtual"` to make comment more clearer